modules/texter.py

```diff
@@ -0,0 +1,7 @@
+# Here is the HSPSMS documentation: https://drive.google.com/drive/u/0/folders/0B3lpvu9ww5atVW9HZUE0b1FBb28
```

Comment:
This is a better link: https://docs.google.com/document/d/1mACLXYFvnjkWJ3oQimN2sscy1G-VGSQAG0zkWhkLYHY/edit?usp=sharing